docs/user/dashboard/lens.asciidoc

```diff
@@ -617,3 +617,34 @@ Pagination in a data table is unsupported. To use pagination in data tables, cre
 ====
 Specifying the color for a single data point, such as a single bar or line, is unsupported.
 ====
+
+[discrete]
+[[dynamic-metric-coloring]]
+.*How does dynamic coloring work for the metric visualization?*
+[%collapsible]
+====
+If you have selected *Value type: Number* in the palette editor, colors will be applied based on the *Primary Metric* value.
+
+The *Primary metric* refers to the large number that is displayed in each tile:
+
+[role="screenshot"]
+image:images/lens_primaryMetric.png[Illustration of where to find the primary metric in a metric visualization.]
+
+
+If you have selected *Value type: Percent*, the primary metric values are mapped to a range between 0 and 100 percent. The bounds of the range depend on your configuration.
+
+The logic is as follows:
+
+If there is a Breakdown dimension (multiple visualization tiles)
+
+* If there is a **Maximum dimension**, the range is from zero to the value of your *Maximum dimension*.
+
+* If there is no *Maximum dimension*, the range is from the smallest of your primary metric values to the greatest of your primary metric values.
+
+If there is no Breakdown dimension (single visualization tile)
+
+* If there is a *Maximum dimension*, the range is from zero to the value of your *Maximum dimension*.
+
+* If there is no *Maximum dimension*, *Value type: Percent* cannot be selected (there's no way for us to determine a range).
```

Comment:
```suggestion
The logic is as follows. If there is a Breakdown dimension for multiple visualization tiles:

* When there is a *Maximum dimension*, the range is from zero to the value of your *Maximum dimension*.

* When there is no *Maximum dimension*, the range is from the smallest primary metric values to the greatest primary metric values.

If there is no Breakdown dimension for a single visualization tile:

* When there is a *Maximum dimension*, the range is from zero to the value of your *Maximum dimension*.

* When there is no *Maximum dimension*, *Value type: Percent* cannot be selected because there's no way to determine a range.
```